packages/contract-wrappers/src/contract_wrappers/contract_wrapper.ts

```diff
@@ -124,39 +112,8 @@ export abstract class ContractWrapper {
         const logWithDecodedArgs = abiDecoder.tryToDecodeLogOrNoop(log);
         return logWithDecodedArgs;
     }
-    protected async _getContractAbiAndAddressFromArtifactsAsync(
-        artifact: ContractArtifact,
-        addressIfExists?: string,
-    ): Promise<[ContractAbi, string]> {
-        let contractAddress: string;
-        if (_.isUndefined(addressIfExists)) {
-            if (_.isUndefined(artifact.networks[this._networkId])) {
-                throw new Error(ContractWrappersError.ContractNotDeployedOnNetwork);
-            }
-            contractAddress = artifact.networks[this._networkId].address.toLowerCase();
-        } else {
-            contractAddress = addressIfExists;
-        }
-        const doesContractExist = await this._web3Wrapper.doesContractExistAtAddressAsync(contractAddress);
-        if (!doesContractExist) {
-            throw new Error(CONTRACT_NAME_TO_NOT_FOUND_ERROR[artifact.contractName]);
-        }
-        const abiAndAddress: [ContractAbi, string] = [artifact.compilerOutput.abi, contractAddress];
-        return abiAndAddress;
-    }
-    protected _getContractAddress(artifact: ContractArtifact, addressIfExists?: string): string {
-        if (_.isUndefined(addressIfExists)) {
-            if (_.isUndefined(artifact.networks[this._networkId])) {
-                throw new Error(ContractWrappersError.ContractNotDeployedOnNetwork);
-            }
-            const contractAddress = artifact.networks[this._networkId].address;
-            if (_.isUndefined(contractAddress)) {
-                throw new Error(CONTRACT_NAME_TO_NOT_FOUND_ERROR[artifact.contractName]);
-            }
-            return contractAddress;
-        } else {
-            return addressIfExists;
-        }
+    protected _getDefaultContractAddresses(): ContractAddresses {
+        return getContractAddressesForNetwork(this._networkId);
```

Comment:
What is `_networkId` is a networkId we do not support? E.g someone of their own testnet 999?